iModelCore/ECPresentation/Source/Rules/CalculatedPropertiesSpecification.cpp

```diff
@@ -116,6 +119,25 @@ bool CalculatedPropertiesSpecification::_ReadJson(BeJsConst json)
     if (json.hasMember(CALCULATED_PROPERTIES_SPECIFICATION_JSON_ATTRIBUTE_EDITOR))
         m_editor = CommonToolsInternal::LoadRuleFromJson<PropertyEditorSpecification>(json[CALCULATED_PROPERTIES_SPECIFICATION_JSON_ATTRIBUTE_EDITOR]);
 
+    if (!json.hasMember(CALCULATED_PROPERTIES_SPECIFICATION_JSON_ATTRIBUTE_EXTENDEDDATA))
```

Comment:
I think this condition should be inverted to `if (has member) { /* do stuff */ }`

Reply:
Changed